applications/mne_scan/plugins/babymeg/babymeg.cpp

```diff
@@ -199,13 +199,6 @@ bool BabyMEG::start()
 
     m_pMyClient->ConnectToBabyMEG();
 
-    if(!m_pFiffInfo) {
```

Comment:
What happens if the fiff info has not been received yet?

Reply:
When we were trying to connect yesterday, it has never been received at this point, it was only appeared set once the data started streaming.

Reply:
The way the code is right now, it is quite difficult to say with certainty. But, as far as I can see, by following the code, if m_pFiffInfo is not set, then the initConnector method would've done nothing which means that the rtmultisamplearray wouldn't have been set properly which means that the run method, basically, does nothing. So the plugin is stable while it leaves time for other plugins to feed info.
But what is clear is that if it ask for fiff info to be there at the beginning it almost-never works (because of timing).